README.md

```diff
@@ -286,15 +288,16 @@ expr 30 \> 20 #1 (true)
 ```bash
 # Number of decimal digit/ significant figure
 echo "scale=2;2/3" | bc
-#.66
+
+$ .66
 
 # Exponent operator
 echo "10^2" | bc
-#100
+$ 100
```

Comment:
should be # instead of $

Reply:
By adding a hashtag you would generate a H1 header and not be demonstrating the shell output of the command. And to stick to your layout for the rest of the document `$` should be the right character here

```bash
echo "10^2" | bc
$ 100 # returned value from the echo command
```